tests/unit_tests/bootstrap.php

```diff
@@ -22,6 +24,7 @@
 
 \VCR\VCR::configure()->enableRequestMatchers(array('method', 'url', 'body'));
 setDummyCredentials();
+\VCR\VCR::configure()->setMode('none');
```

Comment:
Ignore this. This is what I needed to do to get the tests to pass. I have no idea why changing the cache directory would make VCR  go into recording mode (and therefore be unable to authenticate with the fake credentials)